.github/workflows/ci.yml

```
@@ -62,8 +60,7 @@ jobs:
       - name: Install Fuel toolchain
         uses: FuelLabs/action-fuel-toolchain@v0.2.0
         with:
-          name: my-toolchain
-          components: forc@${{ env.FORC_VERSION }}, fuel-core@${{ env.CORE_VERSION }}
+          toolchain: latest
```

You may also use this like so, provided you upgrade to **v0.3.0** ([link](https://github.com/FuelLabs/action-fuel-toolchain#pinned-toolchains)):

```suggestion
          toolchain: latest
          date: 2023-01-18
```

---

This isn't ideal. I was hoping for a decoupling of the apps. This means that if an app needs to be bumped then all the others need to be bumped as well.